### ISSUE TITLE
ADA Enhancements Requested

### ISSUE THREAD

**Issue by paulzav-eh** [Wolox/react-chat-widget#307]
Hi team,

My company recently went through an ADA audit with Essential Accessibility for an application which uses this library, and a handful of issues/enhancement requests pertaining to the html output of this library came to fruition. 

We had to implement a decent amount of javascript to fix some of the ADA issues described above. The problem, is since we are adjusting the html post-render, and we don't have a way to hook into your rendering functions pre-render, the accessibility tree may not have those updates for screen-readers to parse. So, we thought it'd be beneficial to present EA's findings with your team.

- rcw-conversation-container needs a role defined (dialog was suggested)
- would be nice to have a prop to show/hide the rcw-close-button. In the event that it's hidden, would need aria-hidden = true and/or hidden property on it
- the alt text for the title and message avatar images is nondescript - provide an easy way for users to override this alt text
- rcw-input should have an aria-label
- quick buttons should have an aria-label
- rcw-send should have an aria-label
- rcw-launcher should have aria-label and aria-expanded attributes (true if expanded, false if not)
- the aria-live polite setting is not working properly (or is finicky, at best). We have a web service which returns a list of messages, and our application renders each to the user. Even with aria-live polite set as you're setting now, a screenreader interrupts itself (most times...again, it's finicky) and stops reading the old message when a new messages come in.

Thanks and feel free to respond with any questions/comments that you may have.

Paul Z
